ci/tests/ldap/run-ad-server.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
```

Comment:
Could you add a few notes on on how one might go about adding/defining additional accounts or attributes to the samba instance?